system/netdata.logrotate.in

```diff
@@ -0,0 +1,15 @@
+@localstatedir_POST@/log/netdata/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	delaycompress
+	notifempty
+	sharedscripts
+	postrotate
+		if service netdata status > /dev/null ; then \
+			service netdata restart > /dev/null; \
+		fi;
```

Comment:
Ah, OK, got you.

I don't think restart is that bad (and definitely not to get worried about now). e.g. on my system it leaves a gap in the graphs of just 2 seconds.